src/documents/tests/test_api.py

```diff
@@ -3532,6 +3532,68 @@ def test_object_permissions(self):
             status.HTTP_404_NOT_FOUND,
         )
 
+    def test_api_set_permissions(self):
```

Comment:
I know, slightly nit picky, but I do like the given/when stuff

Reply:
Sure thing!